doc/src/yaccextensions.md

```diff
@@ -0,0 +1,14 @@
+# Yacc Extensions
+
+At the beginning of a `.y` file is a `%grmtools{}` section, by default this section is required.
+But a default can be set or forced by using a `YaccKindResolver`.
+
+## Required flags
+    * yacckind
```

Comment:
Missing argument type, and this should list valid values.

Reply:
Should be fixed in [7c21a9d](https://github.com/softdevteam/grmtools/pull/499/commits/7c21a9df013bb73600cdd8c7bd647c943db92cf6)